.github/workflows/ci.yml

```diff
@@ -18,7 +18,7 @@ jobs:
       fail-fast: false
       matrix:
         rust:
-          - 1.30.0
+          - 1.31.0
```

Comment:
I think we need at least 1.37

Reply:
And can you also update the `README.md`? Thanks!

Reply:
Where did you get 1.37 from?